spark/src/main/scala/geotrellis/spark/reproject/TileRDDReproject.scala

```diff
@@ -61,7 +61,8 @@ object TileRDDReproject {
     metadata: TileLayerMetadata[K],
     destCrs: CRS,
     targetLayout: Either[LayoutScheme, LayoutDefinition],
-    options: Options
+    options: Options,
+    partitioner: Option[Partitioner]
```

Comment:
Unrelated to this PR but while reviewing this noticed that `Stitcher` is no longer required constraint here. Might as well remove it in this PR if you don't mind.

Reply:
:+1:

Reply:
It looks like `Stitcher` is still required due to `BufferTiles` https://github.com/locationtech/geotrellis/blob/master/spark/src/main/scala/geotrellis/spark/buffer/BufferTiles.scala#L533-L540

Reply:
Right, but its not needed for the `apply` that already takes RDD of buffered tiles.